service/auth/main_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	"testing"
 )
 
-func TestAuthServer(t *testing.T) {
+func TestAuthServer(_ *testing.T) {
```

Comment:
Test for what ， you should enhance it not ,remove the t

Reply:
Not related to this pr, golint updated and I have to replace `t` to `_`, will fix this latter.

Reply:
seems not necessary.